interface/modules/custom_modules/oe-module-weno/src/Services/LogDataInsert.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ *  @package OpenEMR
+ *  @link    http://www.open-emr.org
+ *  @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @author  Kofi Appiah <kkappiah@medsov.com>
+ *  @copyright Copyright (c) 2020 Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @copyright Copyright (c) 2023 Omega Systems Group International <info@omegasystemsgroup.com>
+ *  @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace OpenEMR\Modules\WenoModule\Services;
+
+use Exception;
+
+class LogDataInsert
+{
+    public function __construct()
+    {
+    }
+    public function insertPrescriptions($insertdata)
+    {
+        $sql = "INSERT INTO prescriptions SET "
```

Comment:
Do not break apart your SQL like this, if you need to use multiple lines, just use a break return. This is practice I see in a lot of code and while _technially_ it works, it is very bad practice.

```php
$sql =  "INSERT INTO prescriptions SET
           active = ?,
           date_added = ?, 
           patient_id = ?, 
           drug = ?, 
           form = ?, 
           quantity = ?, 
           refills = ?, 
           substitute = ?,
           note = ?, 
           rxnorm_drugcode = ?, 
           external_id = ?, 
           indication = ?, 
           start_date = ? ";
```

Reply:
@robertdown, what do you mean by "break return"? can you give an example?

Reply:
Sorry, should have probably said "line return." If splitting over multiple lines it's easier to just start a new line and continue.